llvm/runtimes/CMakeLists.txt

```diff
@@ -432,7 +432,7 @@ if(runtimes)
       # TODO: This is a workaround until flang becomes a first-class project
       # in llvm/CMakeList.txt.  Until then, this line ensures that flang-new is
       # built before "openmp" is built as a runtime project.
-      list(APPEND extra_deps "flang-new")
+      list(APPEND extra_deps "flang-new" "module_files")
```

Comment:
Could you add a comment on why the `module_files` is required here?

Reply:
Done!